### DISCUSSION TITLE
✅ Copy IP address and pass from server list

### DISCUSSION THREAD

**Discussion by sigseg5** [isontheline/pro.webssh.net#836] [Ideas]
Hello, thanks for the awesome app!
I have a lot of SSH servers in WebSSH and I missed really useful feature. How about adding ability to copy IP address of server via right click in server list? Note, edit, clone and delete are not enough I think😄
Thanks a lot!

**Comment by sigseg5** [comment]
And copying the password from the same menu would be very handy

**Reply by isontheline** [reply]
Hello @sigseg5 👋

Thank you so much for your feedback about WebSSH 🙏

I could implement a submenu like this one :
* Copy
  * Hostname
  * Password (will require Touch ID / Face ID while not disabled in settings)

**Reply by sigseg5** [reply]
Thanks a lot❤️

**Reply by isontheline** [reply]
@sigseg5 A first beta release will be available soon (in a few hours max)

**Reply by isontheline** [reply]
@sigseg5 Now available on TestFlight : 
https://testflight.apple.com/join/QSrBK59z
